src/content/json/header-assets.js

```diff
@@ -0,0 +1,49 @@
+import blueprintIcon from "../images/blueprint_banner_negative.webp";
```

Comment:
I don't think this file should be in `src/content/json`. I feel like we could re-factor the src directory to be like:
src
= Header
  = Header.tsx
  = headerAssets.ts
= OtherComponent
  = OtherComponent.tsx
  = otherComponentAssets.ts

OR you could throw all of the styled components into the Header.tsx file directly if you wanted.

Reply:
Do you mean a separate file for each styled component? Feels like overkill for the header 🤔

Reply:
I agree

So what if we just moved all the styled components into the Header.tsx file? B/c I do feel like the styles shouldn't be where it currently is